### PR TITLE
migrate Descope to setup script

### PR DESCRIPTION
**Pull request by nirgur**
fixes: https://github.com/descope/vue-sdk/issues/10

### CODE REVIEW

File: src/Descope.vue [hunk 1]
```diff
@@ -10,79 +11,61 @@
 			:telemetryKey.attr="telemetryKey"
 			:redirect-url="redirectUrl"
 			:auto-focus="autoFocus"
+			:errorTransformer.prop="errorTransformer"
```
Comment (asafshen) [inline src/Descope.vue:14]: Why do we need the .prop suffix here?

**Review by asafshen — APPROVED** [review]
One question

File: example/components/Login.vue [hunk 2]
```diff
@@ -31,7 +31,7 @@ const handleSuccess = (e) => {
 
 const errorTransformer = (error) => {
 	const translationMap = {
-		SAMLStartFailed: 'Failed to start SAML flow'
+		SAMLStartFailed: 'Failed to start SAML flow!!'
```
Comment (dorsha) [inline example/components/Login.vue:34]: I would revert this change.

File: src/Descope.vue [hunk 3]
```diff
@@ -10,79 +11,61 @@
...
+	},
+	autoFocus: {
+		type: Boolean
+	},
+	errorTransformer: {
+		type: Function
 	}
+});
+const emit = defineEmits(['success', 'error']);
+const { projectId, baseUrl, sessionTokenViaCookie } = useOptions();
```
Comment (dorsha) [inline src/Descope.vue:57]: There is a lint issue here, let's make sure this not marked as warning, I would expect CI to be fail in such cases.

Comment (nirgur) [reply src/Descope.vue]: This is for unused attribute, I removed it
Let's merge this PR and change the lint config later on

**Review by dorsha — APPROVED** [review]
